spring-boot-examples/consumer-app/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You are using tabs here, it should be properly formatted like other POM files

Reply:
Is this fixed now? @artur-ciocanu

Reply:
done